init.sh

```diff
@@ -32,13 +32,21 @@ emintd gentx --name $KEY --keyring-backend test
 # Collect genesis tx
 emintd collect-gentxs
 
+# Enable faucet
+cat  $HOME/.emintd/config/genesis.json | jq '.app_state["faucet"]["enable_faucet"]=true' >  $HOME/.emintd/config/tmp_genesis.json && mv $HOME/.emintd/config/tmp_genesis.json $HOME/.emintd/config/genesis.json
+
+echo -e '\n\ntestnet faucet enabled'
+echo -e 'to transfer tokens to your account address use:'
+echo -e "emintcli tx faucet fund 100photon --from $KEY\n"
```

Comment:
does this transfer tokens to your account, or from your account to the faucet?

Reply:
it mints new tokens and transfers them to your account, updating the total supply accordingly

Reply:
do you want me to rename the command from `fund` to `request`?

Reply:
ah okay, yeah that might make more sense. or, you could rename `from` to `to`
but renaming the command to `request` might make more sense, I'm good with either

Reply:
renamed

Reply:
looks good! tested and works